linux_docker_resources/Dockerfile

```diff
@@ -8,17 +8,18 @@ ARG PLATFORM=x86
 # See: http://askubuntu.com/questions/506158/unable-to-initialize-frontend-dialog-when-using-ssh
 ENV DEBIAN_FRONTEND noninteractive
 
+RUN (apt-get update || true) && apt-get install -y locales
```

Comment:
Specifically I'm imagining someone adding a new PPA to the normal Linux machines which then cause apt-get update to fail, but the only thing they see is that their package they expected to find in the PPA isn't installable. They would have seen the issue with apt-get update, but it was being suppressed by the `|| true`.